updater/sponsorships.py

```diff
@@ -54,44 +58,49 @@ def update():
             continue
         for sponsored in sponsoreds:
             context_id = sponsored['args']['addr'].lower()
-
             print('checking sponsored\tapp_name: {0}, context_id: {1}'.format(
                 app['_key'], context_id))
-            context = contexts[app['context']]
-            collection = context['collection']
-            c = db[collection].find(
-                {'contextId': context_id})
-            if c.empty():
-                print("the context id doesn't link to any user under this context")
-                continue
-            user = c.next()['user']
 
             # remove testblocks if exists
             testblocks.delete_match({
```

Comment:
It seems that we can remove this section as the node will apply that when `Sponsor` operation gets received.

Reply:
It's done